react/features/dynamic-branding/middleware.web.js

```diff
@@ -5,7 +5,7 @@ import { MiddlewareRegistry } from '../base/redux';
 
```

Comment:
If you are renaming this file you'll need to adjust the import in middlewares.js

Reply:
I tested this and it worked, I see that we already have it in middewares.web.js. I guess I could either remove the middleware.native.js completely as it is empty, or add it in the middlewares.native.js and it should take the empty one.

Reply:
Shall I just remove middlewares.native.js completely? WDYT?

Reply:
You can remove it then.